cupy/cuda/cufft.pyx

```diff
@@ -57,6 +57,9 @@ cdef extern from 'cupy_cufft.h' nogil:
     Result cufftExecD2Z(Handle plan, Double *idata, DoubleComplex *odata)
     Result cufftExecZ2D(Handle plan, DoubleComplex *idata, Double *odata)
 
+    # Version
+    Result cufftGetVersion(int *version)
```

Comment:
```suggestion
    Result cufftGetVersion(int* version)
```

Reply:
It's a style of CUDA (I copied this from header file) to use `int*`... but I agree with your proposal